src/main/kotlin/io/knotx/distribution.gradle.kts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx
+
+val downloadDir = file("${buildDir}/download")
+val distributionDir = file("${buildDir}/out")
+val stackName = "knotx"
+val stackDistribution = "knotx-stack-${version}.zip"
+val knotxVersion = project.property("knotx.version")
+val isAssembleDistribution = project.hasProperty("knotx.assembleDistribution")
```

Comment:
Please add some comment here. Is `assembleDistribution` the right name - it is more about downloading the stack?

Reply:
hmm. The idea behind is that if only property exists, the build phase will be finalized by `assembleDistribution` - produce full stack zipped. For me the name is right.